<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Modificar</class>
 <widget class="QWidget" name="Modificar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modificar</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="11,2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Nombre:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nombreE"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Categoria:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="categoriaE"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Descripcion:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="descripcionE"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Precio:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="precioE"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">#widget_2{border-top: 5px solid rgb(121, 255, 215) ;}</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="4" column="0">
       <widget class="QPushButton" name="cancelarE">
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="modificarE">
        <property name="text">
         <string>Modificar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
